app/services/fh_services.rb

```diff
@@ -11,15 +11,19 @@ def get_company
     @connection.get "?api-app="+"#{ENV["APPKEY"]}"+"&api-user="+"#{ENV["user_key"]}"
   end
 
-  def get_item
-    @connection.get "marina/items/?api-app=" + "#{ENV["APPKEY"]}"+"&api-user="+"#{ENV["user_key"]}"
+  def get_item(company)
+    if company.class == String
+      @connection.get "#{company}/items/?api-app=" + "#{ENV["APPKEY"]}"+"&api-user="+"#{ENV["user_key"]}"
+    else
+      @connection.get "#{company.shortname}/items/?api-app=" + "#{ENV["APPKEY"]}"+"&api-user="+"#{ENV["user_key"]}"
```

Comment:
Second part of the string is some duplicated code. Also why so many `+`s and not more string interpolation? You're already using it for the env vars.